src/commands/synthetics/run-tests-command.ts

```diff
@@ -171,19 +171,19 @@ export class RunTestsCommand extends Command {
   private config: RunTestsCommandConfig = JSON.parse(JSON.stringify(DEFAULT_COMMAND_CONFIG)) // Deep copy to avoid mutation
 
   public async execute() {
+    const reporters: Reporter[] = [new DefaultReporter(this)]
```

Comment:
@Drarig29 Was there a reason why the reporter was instantiated after this try-catch?

Reply:
No, this must have been a mistake... thanks for catching it 👍